apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -58,6 +58,11 @@
                             "pattern": "^http://[^:\\.\"!#$%^&*()',/]+(\\.[^:\\.\"!#$%^&*()',/]+)*:[0-9]+/$",
                             "example" : "http://somehost.somedomain:123/"
                         },
+                        "listen_address": {
+                          "description": "Listen address for external interface. This should be publicly accessible",
+                          "type": "string",
```

Comment:
Would it be possible to add a pattern for restricting IPv4 address? (As I understand it is what is currently tested.)

Reply:
Wrote you about that on Slack, a working regex for IPv4, IPv6 and potential hostnames is quite complicated. As this is a optional field I am OK with not specifying the pattern here.